documentation/README.md

```diff
@@ -21,3 +21,21 @@ Documentation describing frontend Home Screen Training Video functionality.
 Documentation describing the methods available to developers, for the registration of new custom apps.
 
 - [Registering New Custom Apps](./custom/README.md)
+
+## Frequently Asked Questions
+
+__What's The Deal With Dark Mode In The Samsung Web Browser?__
+
+Although the Samsung browser feature automatically applies it's own dark mode to web pages; it is somewhat restricted, without first seeking the web developer's consent; which results in limited transformational context. Some examples of this, can be seen when attempting to transform images with superimposed text, or small images often used as icons and logos; will tend to get merged with the darker background.
```

Comment:
@kodinkat , @Perumal605 found Samsung phones have a dark mode in their system settings, and a dark mode in their browser settings.

Dark mode works well when dark mode is applied in the browser settings but not when dark mode is applied in system settings. @Perumal605 , would you provide the specific steps to turn off dark mode in system settings and turn on dark theme in the browser settings? I think our writeup should offer turning off system dark mode, and on browser dark mode as a workaround.

Reply:
@incraigulous @kodinkat   First, we need to enable dark mode in the system settings. Once this setting is enabled, open the Samsung Internet browser and go to the menu. There, you will find an option called Light theme sites or Dark theme sites, which can be toggled. After that, navigate to the settings page and select the Labs option. Under Labs, there are two options:

Use system font for webpages
Use website dark theme
The second option is disabled by default, so we need to enable it. These are all the steps.

Reply:
Thanks for this @Perumal605, I'll aim to work these steps into the documentation.